### PR TITLE
typo

### PR DESCRIPTION
**Pull request by Fil**
(spotted by Gordon Tu, thanks!)

### CODE REVIEW

File: docs/reactivity.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ In the future, Plot will support more interaction methods, including brushing. P
 
 The `view` function does two things:
 
-1. it [displays](../javascript#explicit-display) the given DOM *element*, and then
+1. it [displays](<../javascript#display(value)>) the given DOM *element*, and then
```
Comment (mbostock) [inline docs/reactivity.md:298]: Why is this change necessary? Isn’t the original link valid?

https://observablehq.com/framework/javascript#explicit-display

Comment (Fil) [reply docs/reactivity.md:298]: my bad!

**Review by mbostock — APPROVED** [review]
